### DISCUSSION TITLE
Splitting and Converting to Transfer

### DISCUSSION THREAD

**Discussion by BillyMcSkintos** [firefly-iii/firefly-iii#5644] [Firefly III] [answered]
There are a few types of transactions where I need to convert to a transfer and split the transaction E.g. ATM Withdrawal, Mortgage Payment. It looks like a rule can convert to transfer but not a split transaction. 

So, can I convert to transfer when editing the transaction manually? It seems the option to select an existing Asset Account in the Destination account field is missing when _editing_ a transaction (its there when creating a transaction). 

Additionally if I manually create a transaction I can either create BOTH splits as an expense or a transfer. One split cannot be a transfer and the other an expense.

Am I missing something?

ATM Withdrawal transaction is 42.99:
40 Transfer to "Cash Wallet" asset account
2.99 expense to "ATM Fees"

Mortgage Payment of 2000:
1500 transfer to "Home Mortgage" liability (to reduce mortgage amount owed)
500 expense to  the bank for "Mortgage Interest"

**Comment by JC5** [comment, marked as answer]
It won't be possible to create a split using rules, and neither can you create a split with different transaction types. Sorry about that.

**Reply by BillyMcSkintos** [reply]
Understood. Please could splitting with different transaction types be considered a feature enhancement down the road? Thank you.
